src/views/Zap/__tests__/ZapStakeAction.unit.test.tsx

```diff
@@ -164,32 +78,33 @@ describe("<ZapStakeAction/> ", () => {
     expect(await screen.findByText("Successful Zap!"));
   });
 
-  it("Submit Button Should be disabled with < 2 tokens selected enabled with two selected", async () => {
```

Comment:
This tests that the button is disabled until the state is correct. Why remove it?